tests/baselines/pr557_namespacedJsx.baseline.txt

```diff
@@ -0,0 +1,444 @@
+
+const valid3 = <namespaced:Klass.comp.onent namespaced:attr="foo" />;
+
+const invalid1 = <too:much:Namespace />;
+
+const invalid2 = <namespaced-:Klass.Component too:much:namespace-attribute>foo</namespaced-:Klass.Component>;
+
+-----------------------------------
+
+Grammar: TypeScriptReact.tmLanguage
+-----------------------------------
+>const valid1 = <standard-tag standard-attribute="foo" />;
+ ^^^^^
+ source.tsx meta.var.expr.tsx storage.type.tsx
+      ^
+      source.tsx meta.var.expr.tsx
+       ^^^^^^
+       source.tsx meta.var.expr.tsx meta.var-single-variable.expr.tsx meta.definition.variable.tsx variable.other.readwrite.tsx
+             ^
+             source.tsx meta.var.expr.tsx meta.var-single-variable.expr.tsx
+              ^
+              source.tsx meta.var.expr.tsx keyword.operator.assignment.tsx
+               ^
+               source.tsx meta.var.expr.tsx
+                ^
+                source.tsx meta.var.expr.tsx meta.tag.tsx punctuation.definition.tag.begin.tsx
+                 ^^^^^^^^^^^^
+                 source.tsx meta.var.expr.tsx meta.tag.tsx entity.name.tag.tsx support.class.component.tsx
+                             ^
+                             source.tsx meta.var.expr.tsx meta.tag.tsx meta.tag.attributes.tsx
+                              ^^^^^^^^^^^^^^^^^^
+                              source.tsx meta.var.expr.tsx meta.tag.tsx meta.tag.attributes.tsx entity.other.attribute-name.tsx
+                                                ^
+                                                source.tsx meta.var.expr.tsx meta.tag.tsx meta.tag.attributes.tsx keyword.operator.assignment.tsx
+                                                 ^
+                                                 source.tsx meta.var.expr.tsx meta.tag.tsx meta.tag.attributes.tsx string.quoted.double.tsx punctuation.definition.string.begin.tsx
+                                                  ^^^
+                                                  source.tsx meta.var.expr.tsx meta.tag.tsx meta.tag.attributes.tsx string.quoted.double.tsx
+                                                     ^
+                                                     source.tsx meta.var.expr.tsx meta.tag.tsx meta.tag.attributes.tsx string.quoted.double.tsx punctuation.definition.string.end.tsx
+                                                      ^
+                                                      source.tsx meta.var.expr.tsx meta.tag.tsx meta.tag.attributes.tsx
+                                                       ^^
+                                                       source.tsx meta.var.expr.tsx meta.tag.tsx punctuation.definition.tag.end.tsx
+                                                         ^
+                                                         source.tsx punctuation.terminator.statement.tsx
+>
+ ^
+ source.tsx
+>const valid2 = <namespaced:tag namespaced:boolean-attr>foo</namespaced:tag>;
+ ^^^^^
```

Comment:
```:tag``` and ```:boolean-attr``` is getting scoped here instead of just ```tag``` and ```boolean-attr```

Reply:
It was intended, to highlight the colon the same color as the rest of the tag/attribute.
Would you prefer the colon to have its own scope? (`punctuation.separator.namespace.tsx`?)

Reply:
yes please.

Reply:
This should be good